WordPress/Classes/ViewRelated/Post/PostMediaUploadStatusView.swift

```diff
@@ -0,0 +1,103 @@
+import Foundation
+import SwiftUI
+
+/// Displays upload progress for the media for the given post.
+struct PostMediaUploadStatusView: View {
+    @ObservedObject var viewModel: PostMediaUploadViewModel
+    let onCloseTapped: () -> Void
+
+    var body: some View {
+        List {
+            ForEach(viewModel.uploads) {
+                MediaUploadStatusView(viewModel: $0)
+            }
+        }
+        .listStyle(.plain)
+        .toolbar {
+            ToolbarItem(placement: .cancellationAction) {
+                Button(Strings.close, action: onCloseTapped)
+            }
+        }
+        .navigationTitle(Strings.title)
+        .navigationBarTitleDisplayMode(.inline)
+    }
+}
+
+private struct MediaUploadStatusView: View {
+    @ObservedObject var viewModel: MediaUploadViewModel
+
+    var body: some View {
+        HStack(alignment: .center, spacing: 16) {
+            MediaThubmnailImageView(image: viewModel.thumbnail)
+                .aspectRatio(viewModel.thumbnailAspectRatio, contentMode: .fit)
+                .frame(maxHeight: viewModel.thumbnailMaxHeight)
+                .clipped()
+                .frame(width: 40, height: 40)
+
+            VStack(alignment: .leading) {
+                Text(viewModel.title)
+                    .font(.subheadline)
+                    .lineLimit(1)
+                Text(viewModel.details)
+                    .font(.footnote)
+                    .foregroundStyle(.secondary)
+                    .lineLimit(1)
+            }
+
+            Spacer()
+
+            switch viewModel.state {
+            case .uploaded:
+                Image(systemName: "checkmark.circle.fill")
+                    .foregroundStyle(.secondary.opacity(0.5))
```

Comment:
Wdyt of applying a green tint to the check mark icon? I think it makes it more obvious it's done.

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/270ea6fd-502f-4223-80db-cbadb53bcb21" width=200> | <img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/4893d763-8388-401f-8999-3f51fd906a5b" width=200>
-- | --


```suggestion
                    .foregroundColor(Color(uiColor: .success))
```

Reply:
The progress indicators are green, so I wanted something less eye-catching.

Reply:
Hmmm, okay. I see your point - the podcasts app does it too.
<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/df6dc79a-f636-46e7-9b73-d9aba21164d6" width=200>